hera_sim/visibilities/fftvis.py

```diff
@@ -297,12 +290,13 @@ def simulate(self, data_model):
                 beam_spline_opts=data_model.beams.spline_interp_opts,
                 precision=self._precision,
                 polarized=polarized,
+                baselines=antpairs,
                 **self.kwargs,
             )[0]
 
             logger.info("... re-ordering visibilities...")
             self._reorder_vis(
-                req_pols, data_model.uvdata, visfull[:, 0, i], vis, ant_list, polarized
+                req_pols, data_model.uvdata, visfull[:, 0, i], vis, antpairs, polarized
```

Comment:
Hmmm, is this "0" index a hold-over from the pre-future-array-shapes era? I think it is, and if so it should be removed...

Reply:
Yeah, this is due to the future-array-shapes problem. `MatVis` has a similar problem, which is why I chose to leave it this way. I'd be happy to `use_future_array_shapes` in both `FFTVis` and `MatVis`, but that will also require some rewriting of tests. I can either make those changes in this PR or open a separate PR once this is pushed through to make those changes.

Reply:
Ah I see.... yeah let's do this in a different PR. I'll make an issue :-)

Reply:
Sounds good! I'll work on that soon